aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -24,6 +25,7 @@ module aptos_framework::account {
     friend aptos_framework::transaction_validation;
 
     /// Resource representing an account.
+    #[resource_group_member(group=CoreResourceGroup)]
```

Comment:
This would really break things as making an existing standalone resource part of a group means you wouldn't be able to find the existing values due to how the resolver works

Reply:
yeah testing on forge with fresh genesis

sorry didn't intend to publish